lib/Tool/MaintenanceModeHelper.php

```diff
@@ -55,6 +55,9 @@ public function deactivate(): void
     public function isActive(string $matchSessionId = null): bool
     {
         if ($this->db->isConnected() && $maintenanceModeEntry = $this->getEntry()) {
+            if ($matchSessionId === null && $maintenanceModeEntry !== null) {
```

Comment:
```suggestion
            if ($matchSessionId === null) {
```
we can ignore `$maintenanceModeEntry` check as it is already true at this point from line above.